README.md

```diff
@@ -1,2 +1,21 @@
-# product-details
-VTEX Product Details app
+# Product Details
+
+Product Details is a canonical component that any VTEX app can import.
+
+To import it into your code:
+
+```js
+import ProductDetails from 'vtex.product-details/ProductDetails'
```

Comment:
This is not a component that we import. You need to explain how to use as an `ExtensionPoint`. Could you?

Reply:
sure!